### PR TITLE
fix zero division

### PR DESCRIPTION
**Pull request by rajyx**

### CODE REVIEW

**Review by rajyx — COMMENTED** [review]
Done

File: test_common_df_functions.py [hunk 1]
```diff
@@ -40,8 +40,9 @@ def __check_dataframe_metric_delta_column_exists_and_has_correct_value(self, df,
         self.assertTrue(
             f"{metric}_delta" in df.columns
         )
+        count_delta = next_metric_value["count"] - prev_metric_value["count"]
```
Comment (rajyx) [inline test_common_df_functions.py:43]: Done

File: test_common_df_functions.py [hunk 2]
```diff
@@ -40,8 +40,9 @@ def __check_dataframe_metric_delta_column_exists_and_has_correct_value(self, df,
...
         expected_delta = round(
             (next_metric_value["time"] - prev_metric_value["time"]) /
-            (next_metric_value["count"] - prev_metric_value["count"])
```
Comment (rajyx) [inline test_common_df_functions.py:45]: fsdf

File: test_common_df_functions.py [hunk 3]
```diff
@@ -40,8 +40,9 @@ def __check_dataframe_metric_delta_column_exists_and_has_correct_value(self, df,
...
+            (1 if count_delta == 0 else count_delta)
```
Comment (rajyx) [inline test_common_df_functions.py:46]: done

**Review by rajyx — COMMENTED** [review]
done

**Review by rajyx — COMMENTED** [review]
done

File: test_common_df_functions.py [hunk 4]
```diff
@@ -40,8 +40,9 @@ def __check_dataframe_metric_delta_column_exists_and_has_correct_value(self, df,
...
         )
         self.assertTrue(expected_delta == delta)
```
Comment (rajyx) [inline test_common_df_functions.py:48]: done

Comment (gregory-magyx) [inline test_common_df_functions.py:48]: done

**Review by gregory-magyx — COMMENTED** [review]
done

**Review by gregory-magyx — APPROVED** [review]

File: test_common_df_functions.py (hunk 4 above)
Comment (rajyx) [inline test_common_df_functions.py:48]: done

**Review by rajyx — COMMENTED** [review]
Done

File: test_common_df_functions.py (hunk 4 above)
Comment (gregory-magyx) [inline test_common_df_functions.py:48]: done

**Review by gregory-magyx — APPROVED** [review]

File: test_common_df_functions.py (hunk 4 above)
Comment (rajyx) [inline test_common_df_functions.py:48]: adsnekj elairg

Comment (gregory-magyx) [inline test_common_df_functions.py:48]: sdfsdf

**Review by gregory-magyx — APPROVED** [review]

**Review by gregory-magyx — APPROVED** [review]

**Pull request closed without merging**